grails-app/assets/javascripts/managementUnit.js

```diff
@@ -171,6 +171,9 @@ var ManagementUnitPageViewModel = function(props, options) {
     }
     _.extend(self, new ManagementUnitViewModel(props, options));
 
+    var coreReportService = new CoreReportService(options)
+    _.extend(this, coreReportService);
```

Comment:
Best to just delegate to the service instead of extend it.  (Some old services are using this pattern but we should move away from it)